fsutil_test.go

```diff
@@ -106,6 +137,9 @@ func TestIsEmpty(t *testing.T) {
 			t.Errorf("Expected %t, Result was: %t", test.expected, result)
 		}
 	}
+
+	// Remove temp dir
+	os.RemoveAll(dir)
```

Comment:
I changed it. I thought that `t.Error` didn't actually return the function it was more like a print of the error that's why I hadn't used defer